nss-day-cohort-47/tabloid-mvc-cocoapebbles
# cc-tickets 19,21,22

# Description
Added the ability to view, edit, and remove a comment.

## Type of change

# Testing Instructions
To test go into the comment table in your SQL Server Object explorer and manually add a comment into the Comment table. Once that has gone in launch TabloidMVC in the debugger then go to whatever post you added a comment to. Once you are in the post's details go to the far right icon that looks like two chat bubbles and click it. You should be able to view, edit, and delete the comment from that view. 

# Checklist:
- [] I have commented my code, particularly in hard-to-understand areas
